src/decl/grammar.rkt

```diff
@@ -194,6 +196,18 @@
    ]
   )
 
+(define-metafunction formality-decl
+  ;; Find the id of the crate that defines `AdtId`.
```

Comment:
```suggestion
  ;; Find the ID of the crate that defines `AdtId`.
```

Reply:
Huh -- I guess people do capitalize ID, weird. It looks strange to me.